asllib/Tokens.mly

```diff
@@ -33,7 +33,7 @@
 %token LPAR LT MINUS MOD MUL NEQ NOT OF OR OTHERWISE PASS PLUS PLUS_COLON POW
 %token PRAGMA PRINT RBRACE RBRACKET RDIV REAL RECORD RECURSELIMIT REPEAT RETURN
 %token RPAR STAR_COLON SEMI_COLON SETTER SHL SHR SLICING STRING SUBTYPES THEN
-%token THROW TO TRY TYPE UNKNOWN UNREACHABLE UNTIL VAR WHEN WHERE WHILE WITH
```

Comment:
Should we keep UNKNOWN as a reserved word, in case we do want to reuse it for indeterminate values in the future?

Reply:
Makes sense! Will follow up with a new PR for this